packages/augur-ui/src/modules/auth/containers/connect-account.ts

```diff
@@ -4,10 +4,12 @@ import {
   updateAuthStatus,
   IS_CONNECTION_TRAY_OPEN,
 } from 'modules/auth/actions/auth-status';
+import { loadUniverseDetails } from 'modules/universe/actions/load-universe-details';
```

Comment:
I don't think the call should be in connect-acccount

Reply:
We need to have the Universe name to display in the drop-down menu. It isn't initially set. Maybe we should be calling loadUniverseDetails somewhere else to initialize this?

Reply:
hmmm, bummer. we'll we only need to populate the universe info and show a button, if a fork has occurred on the current universe. 

prob add loading in load-forking-info. if a fork has occurred.

Reply:
Makes sense, I'll do that